static/src/javascripts/projects/common/modules/email/email.js

```diff
@@ -210,6 +211,19 @@ define([
                 });
 
             },
+            setTone: function ($el) {
+                var toneClass = 'tonal__main--tone-',
+                    toneClassRegexp = new RegExp('(' + toneClass + ')([a-z-]+)'),
+                    tone;
+
+                if ($el.hasClass('js-email-sub--article')) {
+                    // Get tone from class
+                    tone = $('.tonal__main')[0].className.match(toneClassRegexp)[2];
```

Comment:
Minor comment but I think it's fine to declare a var where it's used rather than explicitly hoisting. Linters can warn you if you use a variable before it is defined (`/*eslint no-use-before-define: 2*/`)